Makefile

```diff
@@ -158,10 +167,33 @@ localnet-start: localnet-stop
 localnet-stop:
 	docker-compose down
 
+setup-contract-tests-data:
+	echo 'Prepare data for the contract tests'
+	rm -rf /tmp/contract_tests ; \
+	mkdir /tmp/contract_tests ; \
+	cp "${GOPATH}/pkg/mod/${SDK_PACK}/client/lcd/swagger-ui/swagger.yaml" /tmp/contract_tests/swagger.yaml ; \
+	./build/gaiad init --home /tmp/contract_tests/.gaiad --chain-id lcd contract-tests ; \
+	tar -xzf lcd_test/testdata/state.tar.gz -C /tmp/contract_tests/
+
+start-gaia: setup-contract-tests-data
+	nohup ./build/gaiad --home /tmp/contract_tests/.gaiad start &
```

Comment:
Ouch.... how are you planning to terminate the process here once tests are completed?

Reply:
nohup is unnecessary I guess, I'll remove it along with his artifacts 👍

Reply:
I tried to suggest a simpler approach with `killall gaiad` do you have concerns regarding this solution @alessio ?

Reply:
I switched to `pkill`, yesterday I tested it on ubuntu and MacOS should be portable now